models/page_views/optional/snowplow_web_ua_parser_context.sql

```diff
@@ -18,7 +18,7 @@ prep AS (
 
     SELECT
 
-      wp.page_view_id,
+      wp.id,
```

Comment:
same here -- I don't believe there's an `id` field in the [snowplow_base_web_page_context](https://github.com/fishtown-analytics/snowplow/blob/master/macros/adapters/default/page_views/snowplow_web_page_context.sql#L34-L35) model?